### ISSUE TITLE
User Dashboard

### ISSUE THREAD

**Issue by stursberg** [hpi-swt2/sport-portal#374]
**This depends on #380 for event logic, #379 for team logic and tile component, #378 for card component.**

AS a user
I WANT to see all my events and teams in one place
IN ORDER to be able to have a clear view and manage all my stuff

Dashboard contains information about:

1. Events that user created or joined (as a player or as a team, where he is a member).
2. Teams that user created or joined.
3. The newest events that were created on platform.
4. (Static) block "Create event".

- reuse card component from #378 
- reuse tile component from #379

Tasks:
- [x] create team tiles
- [ ] create event tiles
- [ ] "create event" button

In wide-screens every block has a 25% of width with paddings/margins.
If width for content is <=800px, then it's only 2 blocks in a row.
In it is <=400 px, then only 1 block in a row. (IT IS ONLY EXAMPLE  - the most important thing, that it looks good on every screen 300 to 1600px).
Don't forget that content has max-width:1200px!

Link "Alle anzeigen" goes to "/events" or "/teams" without any modification of these pages or personalisation. Just static link.

The whole block (pic + name + description) is a link and it goes to corresponding event or team.

Header and Footer are here out of scope!

If user has no events or/and no teams, then "Deine Events" or/and "Deine Teams" will be not shown at all.
"Neuste Events" and "Event create" will be shown always.

It looks like this:
![dashboard.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46ec95892b808ca89dedfd0fe64de29b234324676b5ee17e71d5b07bbd467904580d4bac47585f47f5fcdd7aa4e73a62964e49bc8f3900299f7744eeaa880dbe08b0218e4e83b7b5894bc3e9352de399c2.png)

![db_mobile.png](https://waffleio-direct-uploads-production.s3.amazonaws.com/uploads/59fc457019d4e4001305101f/125516c66e82c728ace21e0d46ec96a52e8d81a083ecfd0fe64de29b234324676b5ee17e71d5b07bbd467904580d4bac47585f47f5fcdd7aa4e73a62964e49bc8f3900299f7744eeaa880dbe06b0218e4e83b7b58944c6ee3527e399c2.png)

**Comment by jkbe** [comment]
**Estimation: M** (but not part of Sprint Backlog)

**Comment by stursberg** [comment]
@emanuel-metzenthin pls merge today till 4pm
